lib/p2p/NodeList.ts

```diff
@@ -49,8 +65,16 @@ class NodeList {
   public load = async (): Promise<void> => {
     const nodes = await this.repository.getNodes();
 
-    nodes.forEach((node) => {
-      this.nodes.set(node.nodePubKey, node);
+    nodes.forEach(async (node) => {
+      if (!node.banned) {
+        this.nodes.set(node.nodePubKey, node);
+        const events = await this.repository.getReputationEvents(node);
+        events.forEach((event) => {
```

Comment:
If you make this `forEach(({ event }) => {` then below you can just use `event` and not the slightly awkward `event.event`.